Linking.md

```diff
@@ -92,7 +91,8 @@ present:
 | Field  | Type             | Description                              |
 | ------ | ---------------- | ---------------------------------------- |
 | offset | `varuint32`      | offset of the value to rewrite           |
```

Comment:
We should move this offset up into the comon header.. not sure why its repeated like this.